docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -182,157 +186,29 @@ spec:
             eviction-hard: 'nodefs.available<0%,nodefs.inodesFree<0%,imagefs.available<0%'
 ```
 
-Copy the above yaml into a local file - clusterclass.yaml - and run:
+The above file can be found in the [CAPI repo](../../../../../test/infrastructure/docker/examples/clusterclass.yaml)
```

Comment:
I'm not sure relative paths works when the site is published (and the relative path is outside the book)